draft-ietf-sipcore-sip-push.xml

```diff
@@ -254,7 +253,17 @@ align="center"><artwork>
         re-registration REGISTER requests. This allows the UA to e.g., use timers for sending REGISTER requests, but 
         to be suspended (in order to save battery resources etc) between sending the REGISTER requests and use push notification to 
         awaken the UA to process incoming calls.
-      </t>      
+      </t>
+      <t>
+        NOTE: This specification does not define any usage of a push notification payload. As defined in  <xref target="section.proxy.req.oth"></xref>,
+        a proxy must not include any payload in the push notification request, and if the associated push notification contains a payload the SIP UA 
+        can simply discard it.
```

Comment:
I still don't understand whether a message with a non-empty payload triggers a REGISTER or not.

Reply:
I will clarify that a message will trigger a REGISTER even if it contains a payload.